.github/workflows/cd.yml

```diff
@@ -104,6 +104,12 @@ jobs:
           then
               git push -d origin $(git tag --points-at HEAD | grep templates | grep rc)
           fi
+            
+      - name: sync up templates deps with sdk version for RC
+        if: ${{ contains(steps.version-change.outputs.CHANGED, '@microsoft/teamsfx') && !contains(steps.version-change.outputs.CHANGED, 'templates@') && github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main' && github.event.inputs.preid == 'rc' }}
+        run: |
+          node ./tools/sync-up-version.js
+        working-directory: ./packages/sdk
```

Comment:
The script should not be put under sdk folder.

Reply:
move to .github/scripts folder